package.json

```diff
@@ -1,13 +1,25 @@
 {
   "name": "tangram-cartocss",
-  "version": "0.6.0",
+  "version": "1.0.0",
   "description": "Transform cartocss into a draw tangram object.",
+  "main": "src/index.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/cartodb/tangram-cartocss.git"
   },
-  "main": "dist/tangram-cartocss.debug.js",
-  "homepage": "https://github.com/cartodb/tangram-cartocss",
+  "dependencies": {
+    "carto": "github:cartodb/carto#dasharrayJSsupport",
```

Comment:
We need to merge that branch before merging this one.

Reply:
Yes